src/main/java/rex/util/Storage.java

```diff
@@ -45,11 +46,12 @@ public void loadFile(TaskList list) throws IOException {
      * @throws IOException If an I/O error occurs while writing to or copying files.
      */
     public void updateFile(TaskList list) throws IOException {
-        File file = new File(filepath);
-        File temp = new File(temppath);
+        File file = new File(FILE_PATH);
+        File temp = new File(TEMP_PATH);
 
-        // Delete the temp file if it exists and create a new temp file
-        Files.deleteIfExists(Paths.get(temppath));
+
+        // Create temp file to copy taskList from

```

Comment:
Would it be better to start with the indicative mood, ie "Creates" rather than "Create" for standardization? I noticed that in several comments it is started with "Create" while others use the indicative mood.

Reply:
Thank you for pointing this out, will make the necessary changes!